### PR TITLE
Routing

### PR DESCRIPTION
**Pull request by gorel**
Here's some more generic changes to the routing tables.

### CODE REVIEW

File: dli_app/mod_wiki/controllers.py [hunk 1]
```diff
@@ -1,20 +1,13 @@
 from flask import (
     Blueprint,
-    flash,
-    g,
-    redirect,
     render_template,
-    request,
-    session,
-    url_for,
```
Comment (ad08fee3) [inline dli_app/mod_wiki/controllers.py:9]: Why did you remove this?

Comment (gorel) [inline dli_app/mod_wiki/controllers.py:9]: We no longer need it.

**Comment by mtracy01** [comment]
hi

**Comment by ad08fee3** [comment]
lgtm

**Comment by gorel** [comment]
https://www.google.com/url?sa=i&rct=j&q=&esrc=s&source=images&cd=&ved=0CAcQjRxqFQoTCMzdvvXQ_scCFYc6kgod1OYE5A&url=http%3A%2F%2Foppositelock.kinja.com%2Flooks-about-right-alternate-headline-sway-bar-supri-1693092171&psig=AFQjCNHoop0c_p1NJxEYJyEs0-3y13vpsA&ust=1442598656183670

**Comment by gorel** [comment]
lgtm

**Comment by mtracy01** [comment]
what

**Comment by XJi** [comment]
275552-baby-seal

**Comment by mozilla2012** [comment]
http://static1.squarespace.com/static/53791236e4b0a0c2854bc164/t/54457f11e4b02e37b41e0e82/1413840657971/
